feature1/one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+This file was also added in the feature1 branch
```

Comment:
Its really good that you have added this line of code

Reply:
Thanks for the comment